tests/publiccloud/prepare_tools.pm

```diff
@@ -31,9 +31,11 @@ sub run {
         }
     }
 
+    # Install prerequesite packages
+    zypper_call('-q in python-xml python3-devel python3-pip python3-img-proof python3-img-proof-tests');
```

Comment:
not so important, but would be useful to print the default python version as we do for the other tools. something like

    record_info('python', script_output('python --version'));

Reply:
Ok, I will add this.